frontend/viewer/src/lib/about/FeedbackDialog.svelte

```diff
@@ -35,7 +35,8 @@
           </div>
         </div>
       </Button>
-      <Button variant="ghost" href={mailtoUrl} class="gap-4 p-4 h-auto text-base justify-start whitespace-normal">
```

Comment:
I just realized that we should probably make the email address more accessible. Maybe:
- Also copy it to the clipboard with a notification?
- Display it in the button text?

mailto links are cool, but they don't necessarily do what a user wants.

Reply:
Yeah I'm not sure. I've noticed that in lexbox our error dialog uses a mailto link, which includes the version, error message etc, but because the address is there they never click it and just copy it and send us an email without any of that helpful info.